src/Fuzzer/Testing/Generators/IInputGenerator.cs

```diff
@@ -0,0 +1,8 @@
+namespace Draco.Fuzzer.Testing.Generators;
+
+internal interface IInputGenerator
```

Comment:
Maybe input generation could be a bit more general, since parsers might just accept tokens as their inputs